exam_2/src/components/mainForm.component.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../actions/fetchWeather'
+import Form from './form.component'
+import Weather from './weather.component'
+
+
+class MainForm extends React.Component {
+    constructor(props) {
+        super(props)
+        this.loadFavs()
+    }
+
+    loadFavs() {
+        const favs = ['Novinki', 'Comrat', 'Sofia'];
+        favs.forEach((fav) => {
+
+            this.props.loadWeather({city: fav}, 'byCity')
+        })
+    }
+
+    render() {
+        this.props.loadWeather()
```

Comment:
```suggestion
      const { loadWeather} = this.props;
      loadWeather()
```
+ вынести в didMount / удалить, если это забытый участок кода